content/en/docs/gke/pipelines/upgrade.md

```diff
@@ -1,134 +1,55 @@
 
-    * Edit `${KF_DIR}/gcp_config/storage-kubeflow.yaml` and set the following
-      flag to skip creating new storage:
+The blueprint pattern makes it easy to upgrade the full Kubeflow in-place, refer to [Update Kubeflow on Google Cloud](docs/gke/deploy/deploy-cli/#update-kubeflow) documentation for instructions.
 
-      ```
-      ...
-      createPipelinePersistentStorage: false
-      ...
-      ```
+However, there's no current support to upgrade from Kubeflow 1.0 or earlier to Kubeflow 1.1 whiling keeping Kubeflow Pipelines data. Provide your feedback in [kubeflow/pipelines#4346](https://github.com/kubeflow/pipelines/issues/4346) if it's important to you.
 
-    * Edit `${KF_DIR}/kustomize/minio/overlays/minioPd/params.env` and specify 
-      the PD that your existing deployment uses for the Minio server:
+## AI Platform Pipelines
 
-        ```
-        ...
-        minioPd=[NAME-OF-ARTIFACT-STORAGE-DISK]
-        ...
-        ```
+Upgrade Support for AI Platform Pipelines is **Alpha**.
 
-    * Edit `${KF_DIR}/kustomize/mysql/overlays/mysqlPd/params.env` and specify 
-      the PD that your existing deployment uses for the MySQL database:
+To upgrade your AI Platform Pipelines instance while keeping existing data:
 
-        ```
-        ...
-        mysqlPd=[NAME-OF-METADATA-STORAGE-DISK]
-        ...
-        ```
+### For instances **without** managed storage:
 
-1. Run the `kfctl apply` command to deploy Kubeflow as usual:
+1. [Delete your AI Platform Pipelines instance](https://cloud.google.com/ai-platform/pipelines/docs/getting-started#clean_up) **without** selecting **Delete cluster**. The persisted artifacts and database data are stored in persistent volumes in the cluster. They are kept by default when you do not delete the cluster.
+1. [Reinstall Kubeflow Pipelines from the Google Cloud Marketplace](https://console.cloud.google.com/marketplace/details/google-cloud-ai-platform/kubeflow-pipelines) using the same **Google Kubernetes Engine cluster**, **namespace**, and **application name**. Persisted data will be automatically picked up during reinstallation.
 
-    ```
-    kfctl apply -V -f ${CONFIG_FILE}
-    ``` 
+### For instances with managed storage:
 
-You should now have a new Kubeflow deployment that uses the same pipelines data 
-storage as your previous deployment. Follow the steps in the deployment guide
-to [check your deployment](/docs/gke/deploy/deploy-cli/#check-your-deployment).
+1. [Delete your AI Platform Pipelines instance](https://cloud.google.com/ai-platform/pipelines/docs/getting-started#clean_up).
+1. If upgrading from 0.5.1, note the Google Cloud Storage (GCS) bucket is a required starting from 1.0.0. Previously deployed instances should be using a bucket named like "<cloudsql instance connection name>-<database prefix or instance name>". Browse [your GCS buckets](https://console.cloud.google.com/storage/browser) to find your existing bucket name and provide it in the next step.
```

Comment:
Suggestion:
- Remember that you're still talking to the user ("you"), add "Kubeflow" to the version number (there are lot of moving parts here)

```diff
- 1. If upgrading from 0.5.1, note the Google Cloud Storage (GCS) bucket is a required starting from 1.0.0. 
+ 1. If you are upgrading from Kubeflow 0.5.1, note that the Google Cloud Storage (GCS) bucket is a required starting from 1.0.0. 
```

Reply:
Nit: 

- Remove "(GCS)"
- Google Cloud Storage -> Cloud Storage
- GCS -> Cloud Storage

Reply:
Thanks!

In fact, 0.5.1 is Kubeflow Pipelines version, I'll add it there explicitly.

@joeliedtke is it the convention to drop Google when the doc is Google Cloud specific? Should I do the same for general purpose docs?

Reply:
That's a good question @Bobgy . I noticed that in Google Cloud docs the first word is dropped and the rest is `Cloud [Product Name (API)]`. Thank you @joeliedtke !